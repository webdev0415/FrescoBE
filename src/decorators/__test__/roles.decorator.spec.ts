import { Roles } from '../roles.decorator';

describe('Roles', () => {
    it('Roles snapshot ', () => {
        expect(Roles).toMatchSnapshot();
    });
});
