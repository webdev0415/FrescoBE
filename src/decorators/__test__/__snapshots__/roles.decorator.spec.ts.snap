// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Roles Roles snapshot  1`] = `[Function]`;
